stock_picking_limit_return_qty/wizard/stock_picking_return.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_stock_return_picking_form_subcontracting" model="ir.ui.view">
+        <field name="name">Return lines</field>
+        <field name="model">stock.return.picking</field>
+        <field name="inherit_id" ref="stock.view_stock_return_picking_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='product_return_moves']/tree/field[@name='quantity']"
```

Comment:
I only see one `quantity` in the view, why not use the simpler _field_ selector ?

```xml
<field name="quantity" position="after" />
        <field name="quantity_max" invisible="1" />
</field>
```
or even
```xml
<form position="inside" />
        <field name="quantity_max" invisible="1" />
</form>

Reply:
Hi @robinkeunen thank you for your feedback! We opted for keeping them more explicit in case this module would be installed on some system which already has some modifications done in the same views.
To make it more capable of survival in a harsh module conditions)

Reply:
Well, I think it's actually more robust to be more generic : `<form>` or `<field name="quantity">` have more chance to survive the future than `//field[@name='product_return_moves']/tree/field[@name='quantity']`.

But ok :-)